pkg/autodiscovery/configresolver/configresolver_test.go

```diff
@@ -86,6 +86,16 @@ func (s *dummyService) GetCheckNames() []string {
 	return s.CheckNames
 }
 
+// IsMetricExcluded returns false
+func (s *dummyService) IsMetricExcluded() bool {
```

Comment:
what do you think if we create only once new method but with one optional parameter

```golang
func (s *dummyService) IsExcluded(types containers.FilterType) bool {
     // ...
}

Reply:
agreed 👍